ebos/equil/equilibrationhelpers.hh

```diff
@@ -825,45 +825,16 @@ double satFromPc(const MaterialLawManager& materialLawManager,
     const PcEq<FluidSystem, MaterialLaw, MaterialLawManager> f(materialLawManager, phase, cell, targetPc);
     double f0 = f(s0);
     double f1 = f(s1);
-
     if (f0 <= 0.0)
         return s0;
     else if (f1 >= 0.0)
         return s1;
-
     assert(f0 > 0 && f1 < 0);
-
-    const int maxIter = 1000;
     const double tol = 1e-10;
-
-    // regula falsi with the "Pegasus" method to avoid stagnation
-    for (int iter = 0; iter < maxIter; ++ iter) {
-        // determine the pivot
-        double s = (s1*f0 - s0*f1)/(f0 - f1);
-
-        // adapt the interval
-        double fs = f(s);
-        if (fs == 0.0)
-            return s;
-        else if ((fs > 0.0) == (f0 > 0.0)) {
-            // update interval and reverse
-            s0 = s1;
-            f0 = f1;
-        }
-        else
-            // "Pegasus" method
-            f0 *= f1/(f1 + fs);
-
-        s1 = s;
-        f1 = fs;
-
-        // check for convergence
-        if (std::abs(s1 - s0) < tol)
-            return (s1 + s0)/2;
-    }
-
-    throw std::runtime_error("Could not find solution for PcEq = 0.0 after "+std::to_string(maxIter)
-                             +" iterations.");
+    const int maxIter = -2*log2(tol)+10;
```

Comment:
If you're still making changes to this file, you may want to consider including `<cmath>` and writing this a bit more explicitly as
```C++
const int maxIter = -2*static_cast<int>(std::log2(tol)) + 10;
```

Reply:
Done.